src/backend/cc.h

```diff
@@ -195,6 +195,7 @@ struct Srcpos
     struct Sfile **Sfilptr;     // file
     #define srcpos_sfile(p)     (**(p).Sfilptr)
     #define srcpos_name(p)      (srcpos_sfile(p).SFname)
+    short Sfilnum;              // file number
```

Comment:
BTW I found it useful to keep all state together, with no intervening declarations. When important I even add comments such as:

```d
struct X
{
    ...  no member variable declarations ...
    // state {
    int a, b, c;
    int* d;
    // }
    ...  no member variable declarations ...
}